pcsx2/gui/MainFrame.cpp

```diff
@@ -643,31 +643,65 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	if( cdvd )
+	if (cdvd2 == nullptr) // It doesn't point to anything when it's explicitly destroyed , let's insert it again.
 	{
-		if( vm )
-		{
-			cdvd->SetItemLabel(_("Reboot CDVD (full)"));
-			cdvd->SetHelp(_("Hard reset of the active VM."));
-		}
-		else
-		{
-			cdvd->SetItemLabel(_("Boot CDVD (full)"));
-			cdvd->SetHelp(_("Boot the VM using the current DVD or Iso source media"));
-		}	
+		// WX throws an assert when a non stock menu item is inserted without a label text as a parameter on Insert()
+		// Let's add a placeholder text just to avoid an assert.
+		cdvd2 = m_menuSys.Insert(1, MenuId_Boot_CDVD2, "Turtleli");
```

Comment:
Update the code to something like that, I liked ``cdvd2 = m_menuSys.Insert(1, MenuId_Boot_CDVD2, "Turtleli");`` better :cry: